src/main/scala/memeid/Constructors.scala

```diff
@@ -42,4 +43,17 @@ trait Constructors {
       })
     }
 
+  // Construct a v4 (random) UUID.
+  def v4[F[_]: Sync]: F[UUID] = Sync[F].delay(new UUID.V4(JUUID.randomUUID))
+
+  def random[F[_]: Sync]: F[UUID] = v4
+
+  // Construct a v4 (random) UUID from the given `msb` and `lsb`.
+  def v4[F[_]: Sync](msb: Long, lsb: Long): F[UUID] = Sync[F].delay {
+    val v4msb = writeByte(mask(4, 12), msb, 0x4)
+    val v4lsb = writeByte(mask(2, 62), lsb, 0x2)
+    new UUID.V4(new JUUID(v4msb, v4lsb))
+  }
+
+  def random[F[_]: Sync](msb: Long, lsb: Long): F[UUID] = v4(msb, lsb)
```

Comment:
I would say to drop this method (it doesn't seem "random" to me).

Reply:
Is an alias of `v4` to generate V4/random UUIDs, I agree is confusing so i'll drop it.